invenio_jobs/assets/semantic-ui/js/invenio_jobs/administration/search/JobRunsHeader.js

```diff
@@ -0,0 +1,75 @@
+// This file is part of Invenio
+// Copyright (C) 2024 CERN.
+//
+// Invenio RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import { RunButton } from "./RunButton";
+import React, { Component } from "react";
+import { NotificationContext } from "@js/invenio_administration";
+import { i18next } from "@translations/invenio_app_rdm/i18next";
+import PropTypes from "prop-types";
+import { http } from "react-invenio-forms";
+
+export class JobRunsHeaderComponent extends Component {
+  constructor(props) {
+    super();
+
+    this.state = {
+      jobId: props.jobId,
+      title: "Job Details",
+      description: "",
+      config: {},
+    };
+  }
+
+  componentDidMount() {
+    const { jobId } = this.state;
+    http
+      .get("/api/jobs/" + jobId)
+      .then((response) => response.data)
+      .then((data) => {
+        this.setState({
+          ...(data.title && { title: data.title }),
+          ...(data.description && { description: data.description }),
+          ...(data.default_args && { config: data.default_args }),
+        });
+      });
+  }
+
+  static contextType = NotificationContext;
+
+  onError = (e) => {
+    const { addNotification } = this.context;
+    addNotification({
+      title: i18next.t("Error"),
+      content: e.message,
+      type: "error",
+    });
+    console.error(e);
+  };
+
+  render() {
+    const { jobId } = this.props;
+    const { title, description, config } = this.state;
+    return (
```

Comment:
I think we miss a loader before the HTTP call finishes

Reply:
As the API call is progressively enhancing the UI, I think showing a loader would make the UI feel laggy and slower than necessary

For context before:
![image](https://github.com/inveniosoftware/invenio-jobs/assets/6676843/2ec26961-3a2f-4d15-a64f-061aa46a49a3)

after:

![image](https://github.com/inveniosoftware/invenio-jobs/assets/6676843/83ed3859-eaf7-4c87-aef8-e8926b5ae9b1)